redhat-developer/developers.redhat.com
# [RHDENG-2655] Ensure Static Resources are hosted by Drupal

This PR ensures that all resources that previously lived within either `/images` or `_docker/lib/export/static` are now directly hosted by Drupal. This is required for when we are running Drupal directly on the Internet as there will no longer be any export process involved to copy the resources into the correct location.

The current export process has been updated to copy the static resources from their new location within the `_docker/drupal/drupal-filesystem` folder.

### JIRA Issue Link
* [RHDENG-2655](https://issues.jboss.org/browse/RHDENG-2655)

### Verification Process

* Visit the Drupal instance (not the export) and ensure that the following can be loaded in your browser
  * /clicktale/ctIframe.htm
  * /googlead73883659545ab5.html
  * /robots.txt
  * /rhdp-apps/onebox/onebox.json
  * A selection of images from the `_docker/drupal/drupal-filesystem/static/images` folder

* Browse the Drupal site and ensure that there are no suspicious looking 404 errors for content as you move through the site
* Now perform the same content checks on the export to ensure that the static resources are being copied correctly by the export process.
